File: example/java/jimmer-sql-graphql/repository/src/main/java/org/babyfish/jimmer/sql/example/repository/BookStoreRepository.java
package org.babyfish.jimmer.sql.example.repository;

import org.babyfish.jimmer.spring.repository.JRepository;
import org.babyfish.jimmer.sql.example.model.BookStore;

import java.util.List;

public interface BookStoreRepository extends JRepository<BookStore, Long> {

    List<BookStore> findByNameLikeOrderByName(String name);
}

File: example/kotlin/jimmer-sql-kt/src/main/kotlin/org/babyfish/jimmer/example/kt/sql/dal/BookStoreRepository.kt
package org.babyfish.jimmer.example.kt.sql.dal

import org.babyfish.jimmer.example.kt.sql.model.BookStore
import org.babyfish.jimmer.spring.repository.KRepository

@JvmDefaultWithCompatibility
interface BookStoreRepository : KRepository<BookStore, Long>